test/exec.js

```diff
@@ -57,7 +61,15 @@ function makeCheck (n, done) {
 // exiting as it once was when the test was designed
 // - [ ] Need test vector or figure out why tail changed
 // Ref: https://github.com/ipfs/js-ipfsd-ctl/pull/160#issuecomment-325669206
-describe.skip('exec', () => {
+// UPDATE: 12/06/2017 - `tail` seems to work fine on all ci systems.
+// I'm leaving it enabled for now. This does need a different approach for windows though.
+describe('exec', () => {
+  // TODO: skip on windows for now
+  // TODO: running under coverage messes up the process hierarchies
+  if (isWindows || process.env['COVERAGE']) {
+    return
```

Comment:
Could this be a proper `.skip()`?

Reply:
`skip` doesn't work outside `it` blocks.